### DISCUSSION TITLE
Softlock

### DISCUSSION THREAD

**Discussion by Wypixla1** [ppy/osu#25424] [Q&A]
### Type

Game behaviour

### Bug description

No matter what, the game was still open, running, (mostly) funtioning, and not crashing, but, the games cursor would stay in place, and only be visible when you had the start menu open, as well as the game not responding to the windows cursor or the keyboard

### Screenshots or videos

https://github.com/ppy/osu/assets/130538048/9249449b-d603-46ea-b1af-19ae3cbc33e1

### Version

2023.1026.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/13343370/database.log)
[input.log](https://github.com/ppy/osu/files/13343371/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/13343372/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/13343374/network.log)
[performance.log](https://github.com/ppy/osu/files/13343375/performance.log)
[runtime.log](https://github.com/ppy/osu/files/13343376/runtime.log)
[updater.log](https://github.com/ppy/osu/files/13343377/updater.log)

**Comment by bdach** [comment]
Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Reply by Wypixla1** [reply]
github won't let me upload it in itself so heres a drive link https://drive.google.com/file/d/1Bi1I0844cnO4zoCZv5ZpaXrUeDg1DUV4/view?usp=sharing

**Reply by bdach** [reply]
- Does this happen with directx renderer? Why are you using opengl?
- Try updating nvidia drivers.

**Reply by Wypixla1** [reply]
I believe OpenGL
